src/main/java/dev/blaauwendraad/masker/json/MaskingState.java

```diff
@@ -73,6 +83,39 @@ public int getReplacementOperationsTotalDifference() {
         return replacementOperationsTotalDifference;
     }
 
+
+    /**
+     * Expands current json path with the key reference
+     */
```

Comment:
I would add `@param`s in JavaDoc and explain what "expand" means in this context

Reply:
Makes sense, I will do it.

Reply:
Done.